internal/kubeapi/fake/fakeclients.go

```diff
@@ -0,0 +1,123 @@
+package fake
+
+/*
+ Copyright 2020 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+
+	"github.com/crunchydata/postgres-operator/internal/config"
+	fakecrunchy "github.com/crunchydata/postgres-operator/pkg/generated/clientset/versioned/fake"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	fakekube "k8s.io/client-go/kubernetes/fake"
```

Comment:
Not a blocker for this PR, but I've noticed we're placing our imports between stdlib and third-party. The convention is to group imports as stdlib, third-party, then "local". `goimports` even has a flag to enforce it:

```
  -local string
        put imports beginning with this string after 3rd-party packages; comma-separated list
```

Reply:
Went ahead and re-arranged per this convention.